packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * Adds OS version support information to README.
```

Comment:
This is part of this release, so needs to be in the 2.1.4 section; see the error message in the CI failure.

Reply:
In case the error message there isn't sufficiently clear, the first six lines of this file after this PR should be:

```
## 2.1.4

* Updates Android Google maps sdk version to `18.0.2`.
* Adds OS version support information to README.

## 2.1.3
```

(The order of the bullet points isn't important, but we usually put the more important parts first when there are multiple items.)